megalodon/src/firefish.ts

```diff
@@ -1303,6 +1303,59 @@ export default class Firefish implements MegalodonInterface {
       .then(res => ({ ...res, data: FirefishAPI.Converter.note(res.data) }))
   }
 
+  	/**
+	 * Convert a Unicode emoji or custom emoji name to a Firefish reaction.
+	 * @see Firefish's reaction-lib.ts
+	 */
+	private reactionName(name: string): string {
+		// See: https://github.com/tc39/proposal-regexp-unicode-property-escapes#matching-emoji
+		const isUnicodeEmoji = /\p{Emoji_Modifier_Base}\p{Emoji_Modifier}?|\p{Emoji_Presentation}|\p{Emoji}\uFE0F/gu.test(name);
+		if (isUnicodeEmoji) {
+			return name;
+		}
+		return `:${name}:`;
+	}
+
+	/**
+	 * POST /api/notes/reactions/create
+	 */
+	public async reactStatus(id: string, name: string): Promise<Response<Entity.Status>> {
```

Comment:
Please use `createEmojiReaction` method: https://github.com/h3poteto/megalodon/blob/950ff623a3a0bdac46f07787a5256a6d200b391d/megalodon/src/megalodon.ts#L1346

Reply:
Also did for deleteEmojiReaction